src/plugins/bfetch/server/plugin.ts

```diff
@@ -20,6 +20,7 @@ import {
 } from '@kbn/core/server';
 import { schema } from '@kbn/config-schema';
 import { map$ } from '@kbn/std';
+import { RouteConfigOptions } from '@kbn/core-http-server';
```

Comment:
nit+question: looks like we can combine this type import with `@kbn/core/server`?

not actually sure which to use now that there are two sources from which the same types can be imported, I guess we should trend toward rather using the types from the package `@kbn/core-http-server`?

Reply:
I will leave it up to the experts on this :) 
I guess once everything migrates to packages, this will be auto resolved?